File: Server/WebApplication1/Models/Job.cs
namespace JobsAPI.Models
{
    // public class JobField {
    //     public string JobField { get; set; }
    //     //  TEACHING , TAXES, PROGRAMMING, KINDERGARDEN 
    //      }
    public class Job

    {
        public int JobId { get; set; }
        public string JobField { get; set; }
        public string JobName { get; set; }
        public string Area { get; set; }
        public string Requirements { get; set; }
        public bool Hybrid { get; set; }
        public int ScopeOfHours { get; set; }
    }
}

File: Client/src/app/models/job.ts
import { JobField } from "./user";
export class job{
    jobId:Number;
    jobField : JobField;
    jobName:string;
    scopeOfHours:number;
    area:string;
    hybrid:boolean;
    requirements:string;

    constructor(jobId:Number,jobField:JobField,jobName:string,requirements:string,scopeOfHours:number,area:string,hybrid:boolean) {
        this.jobId = jobId;
        this.jobName = jobName;
        this.jobField = jobField;
        this.scopeOfHours = scopeOfHours;
        this.hybrid=hybrid;
        this.area=area;
        this.requirements=requirements;
    }
}
